client-next/src/components/MapView/MapView.tsx

```diff
@@ -29,20 +31,33 @@ export function MapView({
   loading: boolean;
 }) {
   const onMapDoubleClick = useMapDoubleClick({ tripQueryVariables, setTripQueryVariables });
-  const [showPopup, setShowPopup] = useState<LngLat | null>(null);
+  const [showContextPopup, setShowContextPopup] = useState<LngLat | null>(null);
+  const [showPropsPopup, setShowPropsPopup] = useState<PopupData | null>(null);
 
   return (
     <div className="map-container below-content">
       <Map
         // @ts-ignore
         mapLib={import('maplibre-gl')}
         // @ts-ignore
-        mapStyle={mapStyle}
+        mapStyle="http://localhost:8080/otp/routers/default/inspector/vectortile/style.json"
```

Comment:
Oh, I just realised that I need to change that.

Reply:
I think you still need to fix this